src/main/clojure/clara/tools/testing_utils.cljc

```diff
@@ -203,3 +203,19 @@
                      e# \newline
                      "Non matches found: " \newline
                      res#)))))))
+
+#?(:clj
+   (defn ex-data-maps
```

Comment:
I've used a slightly modified version of this docstring in https://github.com/cerner/clara-rules/pull/458

Note that I think this can return empty ex-data maps; I have changed the docstring you proposed above accordingly.